src/powerfit_em/powerfitter.py

```diff
@@ -460,14 +455,6 @@ def _allocate_arrays(self):
                             np.zeros(self._ft_shape, dtype=np.complex64))
                         )
 
-        def _build_ffts(self, batch_size=1):
-            self._rfftn = grfftn_builder(self._ctx, self._target.shape,
-                    batch_size=batch_size)
-            self._irfftn = grfftn_builder(self._ctx, self._target.shape,
-                    forward=False, batch_size=batch_size)
-            self._rfftn.bake(self._queue)
```

Comment:
That is a lot less code.

Is correct that self._queue is no longer needed because irfftn/rfftn uses queue of src argument?